src/core/webview/webviewMessageHandler.ts

```diff
@@ -1708,6 +1709,36 @@ export const webviewMessageHandler = async (
 				await provider.toggleTaskFavorite(message.text)
 			}
 			break
+		// kilocode_change start
+		case "fixMermaidSyntax":
+			if (message.text && message.requestId) {
+				try {
```

Comment:
all other calls in this file are set up like this

Reply:
which are probably all vibe coded, but ok.